app/src/main/java/com/account/service/AddRandomSuffixToUsernameService.java

```diff
@@ -0,0 +1,23 @@
+package com.account.service;
+
+import java.util.Random;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class AddRandomSuffixToUsernameService {
```

Comment:
please check naming conventions https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html
Should be smth like `RandomTextGenerator`